src/components/institution/InstitutionAutocomplete.tsx

```diff
@@ -54,12 +57,11 @@ const InstitutionAutocomplete: FC<InstitutionAutocompleteProps> = ({
             ...params.InputProps,
             endAdornment: (
               <>
-                {isLoading && <CircularProgress size={20} />}
+                {isLoading && institutions.length === 0 && <CircularProgress size={20} />}
```

Comment:
vil vi ikke vise progress når man endrer søkestreng (fra et søk som hadde institutions)?

Reply:
er ikke institutions alltid den samme lista her? toplevel? eller brukes denne til underinstitusjoner også?

Reply:
options er jo institutions, og det lastes jo inn uansett om du søker eller ikke? options er prefilled

Reply:
denne brukes for underinstitusjon også ja, ref `InstitutionSelector`. Men ja, de er uansett prefilled de også, så sjekken for length = 0 har kanskje ingen effekt siden enten er isLoading true, eller så har man enten prefilled institutions eller så skal ikke Autocompleten vises i hele tatt

Reply:
problemet er at isLoading ikke rekker å bli satt til false før autocompleten er ferdig-rendret, så den laster hele tiden uten den sjekken